docs/version-bumps.md

```diff
@@ -0,0 +1,42 @@
+# Dependency Updates
+
+This repository receives two kinds of automated pull requests:
+* Dependabot version updates
+* CFN-CI version updates
+
+*Dependabot* is the preferred tool for version updates since it is a well maintained tool provided
+and maintained by GitHub.
+
+*CFN-CI* is our own custom tool to update the dependencies which Dependabot does not know about.
+Primarily those are the dependencies managed via BOSH.
+
+Due to the amount of pull requests generated by these two tools it became a chore to look at them,
+wait for the CI to pass, approve and merge them. Since our main safeguard are the CI checks we
+decided to fully automate those updates:
+
+1. A pull request is opened with a new version.
+2. The `run-ci` label is automatically set.
+3. A GitHub [workflow](/.github/workflows/automerge.yml) checks if the pull request originated from
+   one of the two users, if so the PR is approved and auto-merge is enabled.
```

Comment:
you restricted if actor-confidition in the workflow to dependabot only

Reply:
You mentioned below that it will not work for CFN-CI but I would mention it here that `if the pull request originated from dependabot [...]`